src/screencast/emulatedDevices.ts

```diff
@@ -1,10 +1,293 @@
+

```

Comment:
Do we need this? Also let's add a newline at the end of the file.

Reply:
Nope. That was the EOF newline.